### ISSUE TITLE
Nothing is being shown after using the library

### ISSUE THREAD

**Issue by aaron-imani** [davidsl4/react-telegram-webapp#2]
Hi, thank you for creating this library! I tried to use it on a react app, but it makes the rendered elements disappear! This is the content of my `App.js` file

```
import './App.css';
import Button from 'react-bootstrap/Button';
import Card from 'react-bootstrap/Card';
import profileImage from './13.jpg'
import { useTelegramWebApp, withTelegramWebApp} from 'react-telegram-webapp';

async function validateHash(hash) {
  const response = await fetch(`/api/validate`, {
    method: 'POST',
    headers: {
      'Content-Type': 'application/json',
    },
    body: JSON.stringify({ hash }),
  });

  return response.ok;
}

function App() {
  const app = useTelegramWebApp();
  app.colorScheme="dark";

  return (
      <Card style={{ width: '18rem' }}>
        <Card.Img variant="top" src={profileImage} />
        <Card.Body>
          <Card.Title>Card Title</Card.Title>
          <Card.Text>
            Some quick example text to build on the card title and make up the bulk of
            the card's content.
          </Card.Text>
          <Button variant="primary">Go somewhere</Button>
        </Card.Body>
      </Card>
  );
}

export default withTelegramWebApp(App, {
  validateHash
});
```

**Comment by davidsl4** [comment]
Hi @alirezaimn!
Can you check the browser's console for me? Are there any error messages?

**Comment by aaron-imani** [comment]
<img width="1440" alt="Screen Shot 2022-05-20 at 9 09 04 AM" src="https://user-images.githubusercontent.com/66110688/169557942-450f9702-ac7b-4f96-a683-f1883cf12a04.png">

These are the error messages I receive

**Comment by davidsl4** [comment]
It looks like the validateHash function does not return "true."
Can you please check what the return value is?

**Comment by aaron-imani** [comment]
I don't understand the validateHash function because the URL it tries to access is relevant to the app (You can see that in the screenshot I previously attached).

**Comment by davidsl4** [comment]
You have to implement your validateHash function as stated in the [official Telegram WebApps documentation](https://core.telegram.org/bots/webapps#validating-data-received-via-the-web-app). The hash is validated if the function returns true.

Note that the validateHash function can return a Promise that is resolved into a boolean (true/false) value.

**Comment by J053Fabi0** [comment]
@alirezaimn Did you implemented a valid function? Could you share it, please?

**Comment by J053Fabi0** [comment]
@davidsl4 Even with a function such as `() => true` or `() => Promise.resolve(true)` the same error occurs: `Invalid hash`.

I found and solved the problem in my pull request #3

**Comment by davidsl4** [comment]
@J053Fabi0 thanks!
